changelogs/fragments/51781-fixing-managed-disk-facts.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - azure_rm_managed_disk_facts - added missing implementation of listing managed disks by resource group
```

Comment:
put this into backport rather than devel?

Reply:
@jborean93 mentioned that it's best practice to add changelog fragments with pr in devel already, so I am adding now.